docs/preview/yamlgettingstarted-container.md

```diff
@@ -0,0 +1,141 @@
+
+# YAML getting started - YAML Group Step and Container (not yet available, for discussion only) 
+
+## Group Step
+
+Group step in yaml that represent a set of task steps.
+
+### Syntax
+
+The syntax to declare a group is:
+
+```yaml
+steps:
+- group: string # The group name
+
+  displayName: string
+
+  enabled: true | false
+
+  continueOnError: true | false
+
+  condition: string # Defaults to succeeded(). https://go.microsoft.com/fwlink/?linkid=842996
+
+  timeoutInMinutes: number # Whole numbers only. Zero indicates no timeout.
+  
+  container: string # The container the entire group will runs in.
+  
+  outputs: { string: string} # Output variables mapping for steps within the group, output variables from task in the group wouldn't flow automatically, declare is required. 
+
+  steps: [ script | powershell | bash | task ] # A list of task that this group contains
+```
+
+### Example
+
+A simple build definition with group may look like this:
+
+```yml
+steps:
+- group: group1
+  steps:
+  - script: set
+  - script: "echo ##vso[task.setvariable variable=var3;isoutput=true;]value3"
+    name: groupscript2
+  - script: set
+  outputs: 
+    groupvar1: groupscript2.var3
+    groupvar2: groupscript2.varX
+- script: set    
+```
+
+The group step should fill the gap of the task group concept we had today and make it even more powerful, since the DistributedTask system and agent doesn't understand task group, we have few features are not working as people expected today.
```

Comment:
I would remove this before merging into master. If you agree, I'm wondering whether this type of comment would be more appropriate as a PR comment?